pretty.js

```diff
@@ -46,6 +46,13 @@ function filter (value) {
   return result
 }
 
+function isPinoLine (line) {
+  var requiredProperties = [ 'hostname', 'pid' ]
+  return Object.keys(line).filter(function (k) {
+    return requiredProperties.indexOf(k) > -1
+  }).length > 0
+}
```

Comment:
I have thought about that myself. Before I do it: what do you think of a new common property named `pino`, like the `v` property? It could be set to the current version. Then the check only needs to be for the presence of the `pino` property.